### PR TITLE
Add links to/from d+md pages and rest of site

### PR DESCRIPTION
**Pull request by inglesp**
Addresses #1835.

### CODE REVIEW

File: openprescribing/frontend/tests/test_models.py [hunk 1]
```diff
@@ -249,3 +252,42 @@ def test_regional_team_with_no_pcts_excluded(self):
...
+            possible_savings=10,
+        )
+
+        GenericCodeMapping.objects.create(
+            from_code='0101010I0AAAEAE',
+            to_code='0101010I0AAABAB'
+        )
+
+        # Mag Ox_Cap 100mg
+        self.assertTrue(PPUSaving.objects.exists_for_bnf_code('0101010I0AAABAB'))
```
Comment (sebbacon) [inline openprescribing/frontend/tests/test_models.py]: Can we add a test for the code path where the mapping ends in a percent?

File: openprescribing/dmd2/forms.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 class SearchForm(forms.Form):
     q = forms.CharField(
-        label='Query or SNOMED code',
+        label='Query, SNOMED code, or BNF code/prefix',
```
Comment (sebbacon) [inline openprescribing/dmd2/forms.py]: The BNF prefix will only work from the chemical level, i.e. 9 chars and longer, right?  We need to somehow make this clear - in the label, and perhaps in validation too?

Comment (inglesp) [reply openprescribing/dmd2/forms.py]: > The BNF prefix will only work from the chemical level, i.e. 9 chars and longer, right? 

No -- we just do `Q(bnf_code__startswith=q.upper())` in `DMDObjectQuerySet.search()`.

Currently we ensure that `q` is at least 3 characters long, but maybe it'd be sensible to make it longer.

File: openprescribing/frontend/tests/test_models.py (hunk 1 above)
Comment (inglesp) [reply openprescribing/frontend/tests/test_models.py]: Good spot.

Comment (inglesp) [reply openprescribing/frontend/tests/test_models.py]: No need to do this now.

**Pull request merged**